src/App.js

```diff
@@ -0,0 +1,13 @@
+import Navbar from './components/Navbar';
+import './App.css';
+import Dashboard from './components/Dashboard';
+function App() {
+  return (
+    <>
```

Comment:
instead of a fragment wrap with a div / section for better control of the app. Later we might have usecase for it.